spec/rubocop/cop/rails/skips_model_validations_spec.rb

```diff
@@ -78,4 +78,24 @@
       RUBY
     end
   end
+
+  context 'with whitelist' do
+    let(:cop_config) do
+      {
+        'Blacklist' => %w[toggle! touch],
+        'Whitelist' => %w[touch]
+      }
+    end
+
+    it 'registers an offense for method not in whitelist' do
+      inspect_source('User.toggle!(:attr)')
```

Comment:
Please use the `#expect_offense` matcher as you can see in the examples above. 🙂